### PR TITLE
Simplify preprocessor prompts

### PR DESCRIPTION
**Pull request by AlexJerabek**
This reduces the prompts asked when generating the docs from four to one. It also adds an explanation about creating docs from local files and the review site.

### CODE REVIEW

File: DocumentationToolingNotes.md [hunk 1]
```diff
@@ -24,13 +24,19 @@ There are four relevant d.ts files that provide source content for different sub
...
 
 Older versions of the APIs have their own d.ts files. These are preserved when a new API requirement set is released. They can also be generated using the [Version Remover tool](https://github.com/OfficeDev/office-js-docs-reference/blob/master/generate-docs/tools/VersionRemover.ts). These old d.ts files are maintained so that in the event APIs are patched or altered, the original behavior is still documented. This is useful if you have to target an older version of the API.
 
+#### Locally testing type definition file changes
+
+Any documentation changes for the Office JavaScript API are done by editing the four d.ts files mentioned above. However, you are able to test a change before submitting a PR to DefinitelyTyped (if you need to, for example, test how your formatting will translate into markdown) by editing the corresponding file in [generate-docs/script-inputs](https://github.com/OfficeDev/office-js-docs-reference/tree/master/generate-docs/script-inputs) and running [GenerateDocs.cmd](https://github.com/OfficeDev/office-js-docs-reference/blob/master/generate-docs/GenerateDocs.cmd). When prompted, selected the "Local files" option.
```
Comment (ElizabethSamuel-MSFT) [inline DocumentationToolingNotes.md]:
```suggestion
Any documentation changes for the Office JavaScript API are done by editing the four d.ts files mentioned above. However, you can test a change before submitting a PR to DefinitelyTyped (if you need to, for example, test how your formatting will translate into markdown) by editing the corresponding file in [generate-docs/script-inputs](https://github.com/OfficeDev/office-js-docs-reference/tree/master/generate-docs/script-inputs) and running [GenerateDocs.cmd](https://github.com/OfficeDev/office-js-docs-reference/blob/master/generate-docs/GenerateDocs.cmd). When prompted, select the "Local files" option.
```

File: generate-docs/scripts/preprocessor.ts [hunk 2]
```diff
@@ -10,60 +10,56 @@ tryCatch(async () => {
...
-        //     to avoid being redirected to the EDOG environment on corpnet.
-        // If we ever want to generate not just public d.ts but also "office-with-first-party.d.ts",
-        //     replace the filename.
-    });
 
     console.log('\n');
-    const urlToCopyPreviewOfficeJsFrom = await promptFromList({
-        message: `What is the source of the Office-js TypeScript definition file that should be used to generate the PREVIEW docs?`,
+    const sourceChoice = await promptFromList({
+        message: `What is the source of the Office-js TypeScript definition files that should be used to generate the docs?`,
```
Comment (ElizabethSamuel-MSFT) [inline generate-docs/scripts/preprocessor.ts]: Do we want to note that fallback is DT (like for CDN)?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[bf1080f](https://github.com/OfficeDev/office-js-docs-reference/commits/bf1080f3b9140c3cd07756a3243b225e70e3cd1d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[DocumentationToolingNotes.md](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-SimplifyPreprocessor/DocumentationToolingNotes.md) | :white_check_mark:Succeeded | |
[generate-docs/scripts/preprocessor.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-SimplifyPreprocessor/generate-docs/scripts/preprocessor.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C3%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C202002032125334575-614%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=6vpdTRlkonUzGVAHDIX9xMnWA7XcxViv6R09thFuXCo%3D&st=2020-02-03T21%3A47%3A03Z&se=2020-03-05T21%3A52%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6cc0493](https://github.com/OfficeDev/office-js-docs-reference/commits/6cc04933ff9f6407135979ef51f23b518385f800)_: 

:clock10: **Pending**: waiting for processors ([5](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 5 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

File: generate-docs/scripts/preprocessor.ts (hunk 2 above)
Comment (AlexJerabek) [reply generate-docs/scripts/preprocessor.ts]: The source gets logged later. That seems good enough to debug any potential issues.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6cc0493](https://github.com/OfficeDev/office-js-docs-reference/commits/6cc04933ff9f6407135979ef51f23b518385f800)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[DocumentationToolingNotes.md](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-SimplifyPreprocessor/DocumentationToolingNotes.md) | :white_check_mark:Succeeded | |
[generate-docs/scripts/preprocessor.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-SimplifyPreprocessor/generate-docs/scripts/preprocessor.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C3%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C202002032152391607-614%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=pql3n9WrpI59NO%2FwIsQqBaL1F2R%2BsEu%2BGxI%2BMLlaMG0%3D&st=2020-02-03T22%3A15%3A45Z&se=2020-03-05T22%3A20%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**